kubectl-cloudflow/config/config_test.go

```diff
@@ -585,7 +585,191 @@ func Test_validateConfig(t *testing.T) {
 	}
 	`)
 	assert.Empty(t, validateConfig(okShortLabel, spec))
-	fmt.Printf("okShortLabel: %s\n", validateConfig(okShortLabel, spec))
+
+	volumePodWellformed := newConfig(`
```

Comment:
It's better I think to start writing these as separate tests, we didn't do this really well before, so we should break out of the bad habit.

Reply:
I totally agree with that. I thought about it but didn't go for it (laziness was one of the reasons). One thing I found though is some unexpected errors when running the tests in these new scenarios. For instance, it was complaining it had to have a 'label' or a 'containers.container' after validated de volume mount. I'm saying this because even I agree smaller tests is definitely the way to go I think we will also have to do these ones. Sort of integration if you will.